css/humanPoseAnalyzerSettingsUi.css

```diff
@@ -0,0 +1,121 @@
+#hpa-settings {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1000;
```

Comment:
This z-index should be paired with a corresponding transform: translateZ(1000px)